docs/docs/workflows/trusted-launch.md

```diff
@@ -32,6 +32,14 @@ The script creates the following resources:
 
 Once the import is completed, use the `ID` of the image version in your `constellation-conf.yaml` for the `image` field. Set `confidentialVM` to `false`.
 
+Fetch the image measurements:
+
+```bash
+IMAGE_VERSION=2.0.0
+URL=https://public-edgeless-constellation.s3.us-east-2.amazonaws.com//CommunityGalleries/ConstellationCVM-b3782fa0-0df7-4f2f-963e-fc7fc42663df/Images/constellation/Versions/$IMAGE_VERSION/measurements.yaml
```

Comment:
Shoudn't we do the measurement fetching just before initialization, since then the correct measurements should be featched, since we read out the specified image: https://github.com/edgelesssys/constellation/blob/main/cli/internal/cmd/configfetchmeasurements.go#L118